core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/TableRepairJob.java

```diff
@@ -208,7 +208,8 @@ public boolean runnable()
     public final int getRealPriority()
     {
         RepairStateSnapshot repairStateSnapshot = myRepairState.getSnapshot();
-        int priority = -1;
+        // Default to lastRepairedAt, unless we have repairGroups that we can repair.
```

Comment:
This is not 100% correct either. If we have a repairGroup that can run now, the priority in the queue will be based on that. If a repairGroup becomes available for run a few minutes later then it will have lower priority than it should have locally in the queue.

Reply:
I will need to rethink this, the ideal would be to always have the repairState up to date and base priority on that.